melos.yaml

```diff
@@ -0,0 +1,9 @@
+name: dio
+repository: https://github.com/cfug/dio
+
```

Comment:
Should this have a `publish_to: none`?

Reply:
Does Melos contain the property? Or you're indicating the root `pubspec.yaml`?

Reply:
Ah yea, was thinking about the root pubspec - so we don't accidentally publish the `dio_workspace`.